package.json

```diff
@@ -20,13 +20,16 @@
     "@testing-library/user-event": "14.4.3",
     "@types/jest": "26.0.20",
     "@types/node": "18.19.50",
+    "buffer": "^5.5.0||^6.0.0",
```

Comment:
Why two different versions? It is dev depenendency, not a peer dependency. Why is this needed? Where is this used?